CHANGELOG.md

```diff
@@ -9,14 +9,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+ - Build prp as singularity image from dockerhub in Makefile
+ - Chewbacca and virulencefinder sif fetched from galaxyproject
+ - Added species to amrfinderplus
+ - Added getSpeciesTaxonName to amrfinderplus
+
 ### Fixed
 
-- Fixed config not pointing to the new lowercase repo name: jasen (instead of JASEN)
+ - Fixed config not pointing to the new lowercase repo name: jasen (instead of JASEN)
```

Comment:
Accidental space added? :)

Reply:
Nope. Added one because it wasn't added before.

Reply:
Aha I see that now! Fixing!